docker/rucio_client/scripts/updateDDMQuota

```diff
@@ -90,52 +73,93 @@ def calculate_ddm_quotas():
         if static == 0:
             continue  # Skip if static is 0
 
-        free = static - rucio
-        ddm_quota = STATIC_WEIGHT * (static / total_static) + FREE_WEIGHT * \
-            (free / static) + EXPIRED_WEIGHT * (expired / static)
-        if MAKE_QUADRATIC:
-            ddm_quota = ddm_quota ** 2
-
-        # Override ddm_quota for operational purposes
+        # Control dm_weight for specially configured rses
         rse_attributes = client.list_rse_attributes(rse)
-        if "override_ddm_quota" in rse_attributes:
-            overridden_ddm_quotas[rse] = rse_attributes["override_ddm_quota"]
-            continue
 
-        ddm_quotas[rse] = ddm_quota
-
-
-def normalize_ddm_quotas():
-    weights = [value for value in ddm_quotas.values()]
-    for rse, weight in ddm_quotas.items():
-        ddm_quotas[rse] = int(((weight - min(weights)) / (max(weights) - min(weights))) * MAX_DDM_QUOTA)
+        free = static - rucio
+        dm_weight = static_weight * (static / total_static) + free_weight * \
+            (free / static) + expired_weight * (expired / static)
+        dm_weight_coefficient = 1
+        if "dm_weight_coefficient" in rse_attributes:
+            dm_weight_coefficient = float(rse_attributes["dm_weight_coefficient"])
+        dm_weight *= dm_weight_coefficient
+        if make_quadratic:
+            dm_weight = dm_weight ** 2
+
+        stats[rse] = {
+            "pledge": static,
+            "free": free,
+            "dm_weight_coefficient" : dm_weight_coefficient,
+        }
+
+        dm_weights[rse] = dm_weight
+    return dm_weights
+
+
+def normalize_dm_weights(dm_weights):
+    # TODO: Don't take 0 values into normalization
```

Comment:
I think you did this below.

Reply:
Indeed, thanks for catching it. I'll remove it.